README.md

```
@@ -1,45 +1,406 @@
+  "version": "1.0.0",
+  "wifi_ssid": "HomeNetwork"
+}
+```
+
+</details>
+
+## 🛠️ Development
+
+### Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)